packages/infrastructure/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "location" {
+  type        = string
+  description = "Primary Location"
+  default     = "westeurope"
+}
+
+variable "name" {
+  type        = string
+  description = "name"
+  default     = "alvtime"
+}
+
+variable "env" {
+  type        = string
+  description = "environment"
+}
+
+variable "aks_service_principal_client_id" {
+  type    = string
+  default = "0.0.0.0"
```

Comment:
Could we add descriptions to all of the variables? And could we somehow write a more descriptive description so that other developers can more easily understand what the variables are for?

Reply:
Done